### PR TITLE
culled unnecessary info, moved all the getting started examples to the top

### PR DESCRIPTION
**Pull request by mreinstein**

### CODE REVIEW

**Comment by dblock** [comment]
Hey @mreinstein, you removed a bunch of things that other people do find necessary, like the fact that the docs are for the next version of the library. This is going to waste a lot of time for people who try to figure out why something doesn't work. I value your opinion, but it would have been nice to leave this open for some comments and a healthy discussion by other maintainers.

File: README.md [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
-```
 
-Node.js 0.12 or later is required.
+### AWS Lambda Example
 
-# Stable Release
+Amazon skills that use alexa-app have a built-in `handler` method to handle calls from AWS Lambda.
+You need to make sure that the Handler is set to `index.handler`, which is the default value.
 
-You're reading the documentation for the next release of alexa-app. Please see [CHANGELOG](CHANGELOG.md) and make sure to read [UPGRADING](UPGRADING.md) when upgrading from a previous version. The current stable release is [3.0.0](https://github.com/alexa-js/alexa-app/tree/v3.0.0).
```
Comment (dblock) [inline README.md]: I'd like this to be put back. It links to a document with all the changes and makes it easy to find upgrading notes that are essential for anyone upgrading to a newer version.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,65 @@
...
+  }
+);
 
-The alexa-app module does the dirty work of interpreting the JSON request from the Alexa platform and building the JSON response that can be spoken on an Alexa-compatible device, such as the Echo. It provides a DSL for defining intents, convenience methods to more easily build the response, handle session objects, and add cards.
+// connect the alexa-app to AWS Lambda
+exports.handler = app.lambda();
+```
 
-The intent schema definition and sample utterances can be included in your application's definition, making it very simple to generate hundreds (or thousands!) of sample utterances with a few lines.
+For backwards compatibility, or if you wish to change the Handler mapping to something other than index.handler, you can use the lambda() function.
```
Comment (dblock) [inline README.md]: Lambda should be `quoted`.

**Comment by mreinstein** [comment]
Feel free to revert.

File: README.md [hunk 3]
```diff
@@ -615,34 +584,19 @@ var app = new alexa.app("test");
...
-Use the [alexa-app-server](https://github.com/alexa-js/alexa-app-server) module in combination with alexa-app as a container for multiple alexa-app skills using Node.js and Express. It lets you run and debug your apps locally, and can also be used as a full production server for your apps.
-
-# History
-
-See [CHANGELOG](CHANGELOG.md) for details.
 
-# License
+## License
 
 Copyright (c) 2016 Matt Kruse
```
Comment (dblock) [inline README.md]: Lets update this to 2016-2017.

**Comment by dblock** [comment]
@mreinstein I don't want to revert this, I just want that paragraph back. The other changes look good. Do you agree?

**Comment by mreinstein** [comment]
on a related note, I'm moving the repo i created back under my account (`alexa-verifier`.) I think you should remove me from the alexa-js org because I'm not interested in participating in these interactions.

Thanks for your time, and good luck.

**Comment by dblock** [comment]
Alright. You're being unnecessarily confrontational. Done.

**Comment by mreinstein** [comment]
yes I'm "unnecessarily confrontational" in that I don't want to re-write projects that are essentially done.

**Comment by dblock** [comment]
Thanks for contributing @mreinstein, good luck to you.

**Pull request merged**